apis/python/test/common.py

```diff
@@ -193,16 +214,16 @@ def create_random_dataset_u8(nb, d, nq, k, path):
     return data
 
 
-def create_schema():
+def create_schema(dimension0DomainMax, dimension1DomainMax):
```

Comment:
nit: use variable names with underscores and lowercase

Reply:
Too much C++ 😅 Done.